packages/server/lib/routes.ts

```diff
@@ -98,7 +98,7 @@ export const createRoutes = ({
 
   la(check.unemptyString(config.clientRoute), 'missing client route in config', config)
 
-  app.get(config.clientRoute, (req, res) => {
+  app.get(`${config.clientRoute}`, (req, res) => {
```

Comment:
🤨

Reply:
`config.clientRoute` is typed wrongly - it can be `undefined | null | string | string[]`. This coerces it into a string. 

Should we just do `as string` instead? I guess we could have a `typeof string` check - problem is, if it's not a string, the entire app is gonna break, lol.